commander/src/bootstrapping/templates/lisk-template-ts/templates/app/config/devnet/config.json

```diff
@@ -0,0 +1,2215 @@
+{
+	"networkVersion": "2.0",
+	"label": "devnet",
+	"genesisConfig": {
+		"blockTime": 10,
+		"communityIdentifier": "Lisk",
+		"maxPayloadLength": 15360,
+		"bftThreshold": 68,
+		"minFeePerByte": 1000,
+		"baseFees": [
+			{
+				"moduleID": 5,
+				"assetID": 0,
+				"baseFee": "1000000000"
+			}
+		],
+		"rewards": {
+			"milestones": ["500000000", "400000000", "300000000", "200000000", "100000000"],
+			"offset": 2160,
+			"distance": 3000000
```

Comment:
I think the `config.json` file should be generated separately, so probably can be removed here.